frontend/language/src/nb.json

```diff
@@ -1344,7 +1345,8 @@
   "ux_editor.component_properties.pageRef": "Navnet til siden det gjelder (pageRef)",
   "ux_editor.component_properties.pagination": "Paginering",
   "ux_editor.component_properties.position": "Plassering av valuta",
-  "ux_editor.component_properties.preselectedOptionIndex": "Indeks/plassering av forhåndsvalgt verdi (preselectedOptionIndex)",
+  "ux_editor.component_properties.preselected-help-text": "Må være et tall fra null til og med det antallet valg som finnes",
```

Comment:
If I'm not mistaken the index is 'n-element minus 1,' so the text '... til og med det antallet valg som finnes' is incorrect.
Also hyphen in text key.

Reply:
kanskje noe sånn "Må være et tall fra null til antallet valg minus én)."

Reply:
Ja noe slikt :) Hva tenker du @Ildest ?